Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -280,7 +281,7 @@ class BrowserViewController: UIViewController {
             make.top.equalTo(self.urlBar.snp_bottom)
             make.left.right.equalTo(self.view)
             let url = self.tabManager.selectedTab?.url
-            if url?.absoluteString == HomeURL && self.homePanelIsInline {
+            if url?.absoluteString == HomeURL || self.homePanelIsInline {
```

Comment:
Since we use the same view controller for about:home and the "editing mode" screen, the problem with changing this condition is that we'll end up showing the bottom toolbar in editing mode if the page is on about:home. In other words, if you're on about:home and tap the URL bar, the bottom toolbar should disappear.